llvm/lib/CodeGen/SelectionDAG/SelectionDAG.cpp

```diff
@@ -111,6 +112,10 @@ static void NewSDValueDbgMsg(SDValue V, StringRef Msg, SelectionDAG *G) {
   LLVM_DEBUG(dbgs() << Msg; V.getNode()->dump(G););
 }
 
+STATISTIC(MemopInlined, "Number of memcpy/memmove() calls emitted inline");
+STATISTIC(MemmoveInlined, "Number of memcpy() calls emitted inline");
+STATISTIC(MemcpyInlined, "Number of memcpy() calls emitted inline");
```

Comment:
Not entirely sure whether we need this separate, the MemopInlined is the most useful one.